test/powershell/engine/Module/testmodulerunspace/ModuleWithDependencies2/2.0/ModuleWithDependencies2.psm1

```diff
@@ -0,0 +1 @@
+function Get-ModuleWithDependencies2 { Get-Date }
```

Comment:
I don't see this file used in `ModuleWithDependencies2.psd1`, is it really needed?